services/server/server.go

```diff
@@ -83,6 +91,11 @@ func NewServer(db *reform.DB, prometheus prometheusService, supervisord supervis
 	}
 	path = filepath.Join(path, "pmm-update.json")
 
+	alertsManagerFile := defaultAlertsManagerFile
+	if envAlertFile := os.Getenv("PMM_ALERT_MANAGER_RULES_FILE"); envAlertFile != "" {
```

Comment:
Why don't we use kingpin.Flag like we do for other paths?

Reply:
@percona-csalguero pmm-managed is configured by flags, not environment variables. Let's add a flag, and use it.